src/App.tsx

```diff
@@ -1,18 +1,46 @@
-import React from "react"
-import { Route, Routes } from "react-router-dom"
-const Home = React.lazy(()=> import("./pages/Home/Home"))
+import React, { Suspense } from 'react';
+import { Route, Routes } from 'react-router-dom';
+import { Spinner } from '@chakra-ui/react';
+
+import { useGetSession } from './hooks/useGetSession';
+const Home = React.lazy(() => import('./pages/Home/Home'));
+const Clients = React.lazy(() => import('./pages/Clients/Clients'));
+const Calendar = React.lazy(() => import('./pages/Calendar/Calendar'));
+const Chances = React.lazy(() => import('./pages/Chances/Chances'));
+const Finalized = React.lazy(() => import('./pages/Finalized/Finalized'));
+const Analitics = React.lazy(() => import('./pages/Analitics/Analitics'));
+const LogIn = React.lazy(() => import('./pages/LogIn/LogIn'));
 
 const App = () => {
+const {session} = useGetSession()
+
   const ROUTES = {
```

Comment:
to możesz wrzucić do pliku routes.ts i exportować

Reply:
zrobione